cli/script/command-executor.ts

```diff
@@ -1347,6 +1365,47 @@ function serializeConnectionInfo(accessKey: string, preserveAccessKeyOnLogout: b
     log(`\r\nSuccessfully logged-in. Your session file was written to ${chalk.cyan(configFilePath)}. You can run the ${chalk.cyan("code-push logout")} command at any time to delete this file and terminate your session.\r\n`);
 }
 
+function sessionList(command: cli.ISessionListCommand): Promise<void> {
+    throwForInvalidOutputFormat(command.format);
+
+    return sdk.getAccessKeys()
```

Comment:
What your explaining seems purely based on the historical decisions that we've made, as opposed to what makes the most sense from an end user perspective. The fact that session and access keys are the same is an implementation detail that has zero value to a developer. I don't feel strongly about this, but I don't see the argument for why a consumer of the SDK should know that sessions and access keys are the same.